### PR TITLE
Bug 1303687 - Port templates to DTL

### PR DESCRIPTION
**Pull request by jezdez**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/16)

### CODE REVIEW

File: atmo/templates/account/login.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% extends "account/base.html" %}
```
Comment (maurodoglio) [inline atmo/templates/account/login.html]: This file has probably sneaked in from the other PR of yours, hasn't it?

Comment (jezdez) [inline atmo/templates/account/login.html]: Oh dang, it has, sorry

Comment (jezdez) [inline atmo/templates/account/login.html]: Force-pushed the branch without the file

**Comment by maurodoglio** [comment]
🚢 it !

**Pull request merged**